content/en/blog/_posts/2021-04-08-kubernetes-release-1.21.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: blog
+title: 'Kubernetes 1.21: XXXX'
+date: 2021-04-08
+slug: kubernetes-1-21-release-announcement
+---
+
+**Authors:** [Kubernetes 1.21 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.21/release_team.md)
+
+We’re pleased to announce the release of Kubernetes 1.21, our first release of 2021!
+This release consists of 50 enhancements: 15 enhancements have graduated to stable, 15 enhancements are moving to beta, and 19 enhancements are entering alpha.
```

Comment:
Can we change the number 50 to 49?

We have 2 deprecations, making 51 total enhancements tracked for the release. But, I believe we don't include the 2 deprecations in the total number in the blog post.

Reply:
Done, thank you so much for this!